ui/src/assets/i18n/en.json

```diff
@@ -736,6 +736,7 @@
     "i18n_question_removal_disallowed": "Question cannot be deleted because it is currently attached in published exam.",
     "i18n_action_disabled_lottery_on": "Action is not allowed, because it is part of randomized questions.",
     "i18n_shared_question_property_info": "If you make modifications to this information, it will have an effect to other exams.",
+    "i18n_shared_question_property_info_multi_choice": "If you make modifications to options textinformation, it will have an effect to other exams.",
```

Comment:
Onko tää ihan oikea kännös? Ei vaikuta ihan kieliopillisesti korrektilta.

Reply:
Oisko tää parempi?
The text changes to the options are updated for all exams where the question is used

Reply:
Tää eka siis Helyn antama käännös. 🤷

Reply:
Paha sanoa. Alkuperäisessä options pitäisi kai olla option's ja textinfomation  textual information. Omasta mielestäni :) No laita miten vaan.

Reply:
Eipä ole meidän vastuulla varsinaisesti nämä kuitenkaan.